### PR TITLE
[AD-202] Use LTS-11.2 and cardano-sl's develop

### PR DESCRIPTION
**Pull request by gromakovsky**
I tested it on my machine, didn't have any errors.

### CODE REVIEW

**Comment by gromakovsky** [comment]
Ready for review.

File: ariadne/src/Ariadne/Cardano/Backend.hs [hunk 1]
```diff
@@ -69,25 +66,27 @@ runCardanoNode cardanoContextVar diffusionVar commonArgs sendCardanoEvent = do
...
+              ]
       runNodeReal nodeParams sscParams workers
 
-statusPollingWorker
-  :: (HasConfigurations)
-  => (CardanoEvent -> IO ())
-  -> ([WorkerSpec CardanoMode], OutSpecs)
-statusPollingWorker sendCardanoEvent = first pure $ worker mempty $ \_ -> do
+statusPollingWorker ::
+       (HasConfigurations)
```
Comment (Pastafarianist) [inline ariadne/src/Ariadne/Cardano/Backend.hs]: I thought reflection was removed from `cardano-sl/develop`?

File: ariadne/src/Ariadne/Wallet/Cardano/Kernel.hs [hunk 2]
```diff
@@ -30,26 +30,27 @@ module Ariadne.Wallet.Cardano.Kernel (
   , hasPending
   ) where
 
-import           Universum hiding (State)
+import Universum hiding (State, init)
```
Comment (Pastafarianist) [inline ariadne/src/Ariadne/Wallet/Cardano/Kernel.hs]: Please avoid applying `stylish-haskell` in this part of the code, or use the Cardano config. We'd like to make import of upstream changes as seamless as possible.

File: stack.yaml [hunk 3]
```diff
@@ -54,17 +54,13 @@ packages:
...
-    commit: 9a8af2440d655e14b802639b0b363be2ffb5a32a
-  extra-dep: true
 - location:
     git: https://github.com/serokell/kademlia.git
     commit: 7120bb4d28e708acd52dfd61d3dca7914fac7d7f # master
   extra-dep: true
 - location:
-    git: https://github.com/input-output-hk/plutus-prototype
-    commit: d4aa461fc69fc6957aab46b41a670c2144aefb77
+    git: https://github.com/avieth/plutus-prototype
```
Comment (Pastafarianist) [inline stack.yaml]: :tada:

File: ariadne/src/Ariadne/Wallet/Cardano/Kernel.hs (hunk 2 above)
Comment (maksbotan) [reply ariadne/src/Ariadne/Wallet/Cardano/Kernel.hs]: Really? As I've seen, imported code is restyled with stylish-haskell config from ariadne repo. My script does that, at least.

Comment (Pastafarianist) [reply ariadne/src/Ariadne/Wallet/Cardano/Kernel.hs]: You are right, I somehow thought it wasn't. In that case please disregard my comment, and I wonder how this file escaped restyling.

File: ariadne/src/Ariadne/Cardano/Backend.hs (hunk 1 above)
Comment (gromakovsky) [reply ariadne/src/Ariadne/Cardano/Backend.hs]: Not fully.

File: ariadne/src/Ariadne/Wallet/Backend/Mode.hs [hunk 4]
```diff
@@ -21,21 +21,19 @@ import Pos.Core (Address, largestHDAddressBoot)
...
-import Pos.Util.CompileInfo (HasCompileInfo)
 
 import Ariadne.Cardano.Face (CardanoMode)
 
 instance HasConfiguration => MonadBalances CardanoMode where
     getOwnUtxos addrs = getFilteredUtxo addrs
     getBalance = getBalanceFromUtxo
 
-instance (HasConfigurations, HasCompileInfo) => MonadTxHistory CardanoMode where
+instance (HasConfigurations) => MonadTxHistory CardanoMode where
```
Comment (hithroc) [inline ariadne/src/Ariadne/Wallet/Backend/Mode.hs]: These parenthesis are redundant now

**Pull request merged**